src/components/Info/Form.js

```diff
@@ -118,33 +132,33 @@ export default ({ onSubmit, city, setError, error, setOpen, setSubmitted }) => {
                                                />
                                                <span>I am taking a plus one</span>
                                         </label>*/}
-      {plusOne && (
+      {formState.plusOne && (
```

Comment:
I think this would look cleaner if the necessary variables for `formState` are restructured around line 25.